src/routes/users.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const router = new express.Router();
+const User = require('../models/user');
+const Users = require('../controllers/users');
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
+  res.send('respond with users');
+});
+
+// TODO: Role protect?
+router.get('/all', async function(req, res, next) {
```

Comment:
perm levels can be achieved with a middleware.
[shamelessly pointing to my repo again](https://github.com/billhanyu/blog/blob/master/server/authMiddleware.js)